app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    def remove_from_alive(self) -> None:
+        Animal.alive.remove(self)
```

Comment:
I think mentor will reject this code, because you don't need this method inside class. You call this only one time.

Reply:
> I think mentor will reject this code, because you don't need this method inside class. You call this only one time.

I think there is an OOP principle that states the following: 

Encapsulation is one of the fundamental concepts in Object-Oriented Programming (OOP) that involves bundling data (attributes) and methods (functions) that operate on that data into a single unit, known as a class. 

That's why this method, in my opinion, should be inside the class.

Reply:
That's all right, because Animal is an abstraction that contains the alive ancestors